### PR TITLE
[RFC] cancel_squad_testjobs: Add --dry-run option

### PR DESCRIPTION
**Pull request by mrchapp**

### CODE REVIEW

File: bin/cancel_squad_testjobs.py [hunk 1]
```diff
@@ -8,7 +8,13 @@
 import squad_client  # noqa: E402
 
 
-def cancel_lava_jobs(url, project, build_version, identity=None):
+class RuntimeConfig(object):
```
Comment (danrue) [inline bin/cancel_squad_testjobs.py:11]: Maybe it's a good idea, but I don't see the benefit to making a class to hold two variables which are only used in one function. Seems to add complexity - what's the benefit?

Comment (mrchapp) [reply bin/cancel_squad_testjobs.py:11]: Thanks, that was exactly the kind of comments I expected and the questions I was pondering on. The alternative would be to add `dryrun` to the list of arguments of `cancel_lava_jobs()`. It's just one argument, not a big deal, but still felt odd adding stuff like that.

Here's the alternative: https://github.com/Linaro/lkft-tools/pull/30

Comment (danrue) [reply bin/cancel_squad_testjobs.py:11]: I think #30 is more correct. The thing is here, you're not removing an option, you're just finding another place to put it and adding complexity/misdirection with no benefit. If RuntimeConfig was reusable in multiple places, maybe it makes sense - though i'd still be tempted to create a class out of cancel_lava_jobs instead.

Comment (mrchapp) [reply bin/cancel_squad_testjobs.py:11]: It's settled, then.

**Comment by mrchapp** [comment]
Closing in favor of #30.

**Pull request closed without merging**